ssg/build_remediations.py

```diff
@@ -146,6 +151,81 @@ def get_populate_replacement(remediation_type, text):
     sys.exit(1)
 
 
+def from_file(file_path, env_yaml):
```

Comment:
This name is really not descriptive.

Reply:
Would adding a descriptive doc string here be acceptable instead of renaming the function?

I'm partial to functions which aren't repetitive. The fully qualified name to this function is `ssg.build_remediations.from_file(...)`, which I believe makes sense. Most of the time, we call `import ssg.build_remediations as remediation` (`build-scripts/combine_remediations.py#L11`), so the qualified name is `remediation.from_file(...)` which also makes sense.

Reply:
I've also converted the return type to a `namedtuple` so that should be more obvious as well.

Reply:
I see your point. I think that e.g. `parse_from_file` won't ruin the fully qualified name, while it hints that some processing of the text and maybe even metadata extraction may take place.

On an unrelated note, I prefer function names over docstrings because docstrings may become obsolete and consequently misleading before you know it.